include/seqan/modifier/modifier_shortcuts.h

```diff
@@ -200,13 +200,13 @@ typedef ReverseComplementString<Rna5String>::Type    Rna5StringReverseComplement
  */
 
 template <typename TSequence >
-inline void complement(TSequence & sequence)
+inline SEQAN_FUNC_ENABLE_IF(IsSequence<TSequence>, void) complement(TSequence & sequence)
```

Comment:
Don't you want to fix the stringset overload below also?

Reply:
The StringSet overload is already constrained to StringSets so it's not a problem!

Reply:
You sure that stringsets are physically restricted to contain sequences? :sweat_smile:

Reply:
But what about `reverseComplement(TSequence & sequence, Tag<TParallelTag> parallelTag)`? :stuck_out_tongue_closed_eyes:

Reply:
What are you talking about? This is just about preventing the overload from being used for e.g. `int`.